src/nvim/auevents.lua

```diff
@@ -29,6 +29,7 @@ return {
     'CmdWinLeave',            -- before leaving the cmdline window
     'ColorScheme',            -- after loading a colorscheme
     'ColorSchemePre',         -- before loading a colorscheme
+    'MenuPopupChanged',              -- after puopup menu drawn to screen
```

Comment:
list must be alphabetical

Reply:
this still looks not alphabetical. it should be listed after MenuPopup, not ColorSchemePre.

Reply:
Fixed.